content/v2.0/influxdb-templates/create.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Create an InfluxDB template
+description: >
+  Use the InfluxDB UI and the `influx pkg export` command to create InfluxDB templates.
+menu:
+  v2_0:
+    parent: InfluxDB templates
+    name: Create a template
+    identifier: Create an InfluxDB template
+weight: 101
+v2.0/tags: [templates]
+---
+
+Use the InfluxDB user interface (UI) and the `influx pkg export` command to
+create InfluxDB templates.
+Add resources (buckets, Telegraf configurations, tasks, and more) in the InfluxDB
+UI and export then the resources as a template.
+
+{{% note %}}
+Templatable resources are scoped to a single organization, so the simplest way to create a
+template is to create a new organization, build the template within the organization,
+then [export all resources](#export-all-resources) as a template.
+
+**InfluxDB OSS** supports multiple organizations so you can create new organizations
+for the sole purpose of building and maintaining a template.
+In **InfluxDB Cloud**, your user account is an organization.
+**We recommend using InfluxDB OSS to create InfluxDB templates.**
+{{% /note %}}
```

Comment:
Do we recommend using InfluxDB OSS to create InfluxDB templates solely because Cloud only has one organization?

Reply:
Yes. What we don't want happening is users creating new cloud accounts solely to create templates. That will skew our numbers.